src/core/server/logging/appenders/file/file_appender.ts

```diff
@@ -73,4 +75,8 @@ export class FileAppender implements DisposableAppender {
       });
     });
   }
+
+  private ensureDirectory(path: string) {
+    mkdirSync(dirname(path), { recursive: true });
```

Comment:
I'm not proud of the sync mkdir here, but I'm not sure theres any other way to do this if it happens in the `append` step? At least this is just a rare thing only when `outputStream` is `undefined`, so probably not any issue if understand the flow correctly